tests/unit/test_generate_umm_var.py

```diff
@@ -1,9 +1,6 @@
-from os.path import exists
-from shutil import copy, rmtree
-from tempfile import TemporaryDirectory
+from shutil import copy
 from unittest import TestCase
 from unittest.mock import ANY, patch
-import json
```

Comment:
This is `ruff` removing unused imports in this test module. I ran the tests locally before pushing and they still passed. The CI/CD for this PR also looks good, so I'm comfortable with this change.